### ISSUE TITLE
NTR: vesicle assembly

### ISSUE THREAD

**Issue by BarbaraCzub** [geneontology/go-ontology#13907]
Figure 1 in PMID:25898166 shows that knock down of PICALM Q13492 in cultured cells results in an increase in size of clathrin-coated vesicles and in reduced ability to 'close' the vesicles being formed.

I would like to capture this using a new term 'vesicle maturation', which could be defined as:
"A process that is carried out at the cellular level, which results in the assembly and arrangement of constituent parts of a vesicle. Vesicle maturation begins with membrane bending (GO:0097753) and ends with vesicle scission (GO:0099050)."

> GO:0016050 vesicle organization
>> NTR vesicle maturation
>>> GO:0016188 synaptic vesicle maturation

I also wonder whether the definition of 'GO:0016188 synaptic vesicle maturation' should be updated to include initial and final step as well?

Please use refs: GOC:aruk and GOC:bc.

This is related to #13906.

Thanks,
Barbara

**Comment by ValWood** [comment]
Isn't this also  just a phenotypic consequence of a  a pathological defect in GO:0048268 clathrin coat assembly?  (i.e problems recruiting clathrin). It seems like a phenotype rather than a real process of maturation?

**Comment by BarbaraCzub** [comment]
But isn't 'GO:0048268 clathrin coat assembly' a part of the broader process of 'vesicle maturation'?

**Comment by ValWood** [comment]
That might be sensible if the process of vesicle formation is broken down  and there are specific maturation stapes after coat assembly. That I am not sure about. 

This term

synaptic vesicle maturation
--
Ontology | Biological Process
Definition | Steps required to form an initiated synaptic vesicle into a fully formed and transmissible synaptic vesicle.


is not very enlightening about which parts of the process are considered to be maturation, so it's unclear. Do you know which steps ?

**Comment by ValWood** [comment]
synaptic vesicle assembly has one descendant
GO:0097401 synaptic vesicle lumen acidification
classed as maturation.

**Comment by BarbaraCzub** [comment]
hm... I've just tried to google 'vesicle maturation' and it seems that biologists do not generally use this phrase. So now that I'm thinking about this, and looking at the definition, which I suggested ("A process that is carried out at the cellular level, which results in the assembly and arrangement of constituent parts of a vesicle. Vesicle maturation begins with membrane bending (GO:0097753) and ends with vesicle scission (GO:0099050)."), I suppose what I actually meant to request was 'vesicle assembly', so that it would only refer to the structural aspects of a vesicle. 
On second thought I suppose it would be impossible to define an endpoint for 'maturation' as vesicles are not constant structures, but they transition gradually from one into another, as their pH and other characteristics change. E.g. a 'very mature' late endosome would probably be biochemically indistinguishable from an 'early maturating' lysosome...
Perhaps in case of the 'synaptic vesicle maturation', it is a bit easier to justify the existence of this term (although the definition certainly requires more detail), as it represents only one vesicle 'type', which gradually becomes more and more acidic, and so 'mature' (?). Whereas for the 'default' vesicles, the less acidic endosome eventually 'becomes' the more acidic 'lysosome', so the process of lumen acidification results in generation of different vesicle 'types' (even though the mechanism is virtually the same as for changing an 'immature synaptic vesicle' into a 'mature synaptic vesicle').

**Comment by ValWood** [comment]
I agree with you. I think it would be difficult to define a beginning and end of maturation, unless the steps it refers to are know. 
It might be worth asking @dosumis about the placement of 
GO:0097401 synaptic vesicle lumen acidification
under "synaptic vesicle maturation"

The use of "maturation" here seems too vague.


v

**Comment by BarbaraCzub** [comment]
Hi @hdrabkin please disregard my earlier request for the 'vesicle maturation' term.

I have just discussed this with @RLovering and I wish to change my new term request.

I am now requesting a new term 'vesicle assembly':

> GO:0016050 vesicle organization
>> NTR vesicle assembly

_Suggested_ definition: "A process that is carried out at the cellular level, which results in the arrangement of constituent parts of a vesicle. Vesicle assembly begins with membrane bending (GO:0097753) and ends with vesicle scission (GO:0099050)."

_Suggested_ synonyms:
vesicle formation
vesicle biosynthesis

Please use refs: GOC:aruk and GOC:bc.

Thanks,
Barbara

**Comment by BarbaraCzub** [comment]
Please see also #13915.

**Comment by dosumis** [comment]
Isn't endocytosis a type of vesicle assembly by this definition?  What about membrane budding?  (Looking at these two again, I think endocytosis should be at type of 'vesicle budding', rather than having it as a part).

**Comment by dosumis** [comment]
Also looks like 'vesicle scission' should be part of membrane budding.

**Comment by BarbaraCzub** [comment]
Yes, I think it makes sense for 'vesicle scission' to be a part of 'membrane budding'.

**Comment by BarbaraCzub** [comment]
Wrt to endocytosis, the process will indeed begin with 'vesicle assembly', but not all newly assembled vesicles will be endocytic vesicles.

**Comment by hdrabkin** [comment]
NTRs added

**Comment by BarbaraCzub** [comment]
Thank you @hdrabkin 
Although I am now not convinced about the second one now: 'NTR2 vesicle fusion involved in vesicle assembly' because @dosumis clarified (in #13915) that 'fusion' should only be used for two membranes getting joined together, whereas in this case it would be one membrane 'closing/sealing up'. So perhaps let's put NTR2 on hold for now, as @dosumis suggested this process of membrane closing up does in fact happen during 'vesicle scission (GO:0099050)' #13915, so it is this latter term that probably needs modifying?

**Comment by ValWood** [comment]
What do we really mean by vesicle assembly though. Its odd after all of the work in cerevisie that such a term has not been requested before....

I would say that a vesicle is formed as a result of a set of processes (vesicle formation might be better than assembly, although some parts are assembly).

For example terms like
O:0099049 clathrin coat assembly involved in endocytosis
are defined as
The process that results in the assembly of clathrin triskelia into a clathrin cage during endocytosis. Clathrin is recruited to the plasma membrane via interaction with scaffolding proteins that bridge between clathtin and cell surface receptors. Clathrin coat formation is concomittant with coated pit formation leading to endocytic vesicle formation.

Perhaps it is just  vesicle  formation  is worked out in quite mechanistic detail so nobody though to add an explicit parent?

but there are  also lots of existing terms related to these processes, for example most of the terms under
GO:0006901 vesicle coating

**Comment by hdrabkin** [comment]
It's too late, as I have already added the term. So it would need to be obsoleted.

**Comment by dosumis** [comment]
I'm with Val on this.  Sorry for not being clearer before.  If vesicles are always formed by budding from membranes, budding includes invagination, and scission and we have various terms for budding, then why do we need terms for vesicle assembly? What additional processes come under assembly that are true for all types of vesicles?

**Comment by BarbaraCzub** [comment]
Now that I've looked at the term 'GO:0006900 membrane budding' more closely I see that it has the synonyms 'vesicle formation/assembly/biosynthesis', and the definition does in fact say that it results in vesicle formation. 

But is it accurate to say that membrane budding is synonymous with vesicle formation/assembly? 

( @ValWood , @RLovering and I discussed 'assembly' vs. 'formation', and having looked at existing terms decided that the two have been used interchangeably in GO, but with 'assembly' typically being used as the primary term). 

Membrane budding needs to occur in order for a vesicle to form, but the scission also needs to occur, so ultimately I think both would form a part of the broader process of membrane formation/assembly. That is, unless, a 'vesicle' is defined as a structure, which can exist while being attached to another membrane (which would be the case before scission occurs). However, the definition of 'GO:0031982 vesicle', which states that is is 'Any small, fluid-filled, _spherical_ organelle _enclosed by membrane_,' suggests that it is a 'vesicle' only after scission has occurred, or else it could not be _spherical_, or _enclosed by membrane_. Or unless 'vesicle scission' becomes a child of 'membrane budding' as @dosumis suggested above, and which I originally agreed with, but upon reflection I do not think this is a good solution.

Having said that, I now see that I can use the (more specific) term 'GO:0006900 membrane budding' to capture what PICALM is shown to do in PMID:25898166. Thank you for your helpful comments. 

Still, the definition of 'membrane budding' needs to be updated. And I suppose it should also be a child of 'GO:0097753 membrane bending'? 

Wrt to the NTR, it is probably worth having the parent 'vesicle assembly' (or 'vesicle formation' - I think either one would work as the primary term), which would encompass membrane bending, budding and vesicle scission. And as @ValWood pointed out, more specific terms, such as 'GO:0099049 clathrin coat assembly involved in endocytosis', should probably also then become child terms of 'vesicle formation'. (The current overall parent term 'GO:0016050 vesicle organization' covers both: assembly as well as disassembly). 

Going back to 'vesicle formation/assembly' @hdrabkin perhaps the definition, which I suggested, could be updated not to include the word 'fusion' as @dosumis explained, but e.g. 'closure' instead: "A process that is carried out at the cellular level, which results in the arrangement of constituent parts of a vesicle. Vesicle assembly begins with membrane bending (GO:0097753) and ends with closure of the vesicle during vesicle scission (GO:0099050)"?

**Comment by hdrabkin** [comment]
Ok, I can I obsolete the NTR2 vesicle fusion involved in vesicle assembly, leaving NTR1? Since I just made it yesterday, I don't think I need to go through the standard emails, etc.,

**Comment by dosumis** [comment]
>  I can I obsolete the NTR2 vesicle fusion involved in vesicle assembly
That should definitely go.

I'm still not really clear why budding (with scission as a part) is not sufficient.   

If you really want a new parent class, then think you should look at subclasses of 'membrane budding' and think about whether we need new terms there too.   Also, assembly feels like a very odd choice of name in this case as it brings to mind a process where various components are brought together to make a structure *de novo*.  'vesicle formation' would be better.

**Comment by ValWood** [comment]
I also think the current terms worked...

**Comment by dosumis** [comment]
Perhaps if you rename 'membrane budding' to  'vesicle budding from membrane' the ontology would be clearer.  Lots of coating terms are already a part of membrane budding.   If membrane budding ('vesicle budding from membrane') remains the general class covering vesicle formation, there should probably be some mention of coating in the def (where coating of a membrane is necessary to invagination, this is considered part of the process of budding'.

At the risk of confusing matters, here's a relevant monster ticket
https://github.com/geneontology/go-ontology/issues/12403#issuecomment-235905631 
(Almost everything on this one done, but couple of remaining issues)

**Comment by RLovering** [comment]
I would like to see 'membrane budding' renamed to 'vesicle budding from membrane'. There is at least one other occurance of membrane budding which does not result in vesicle formation and that is the formation of enveloped viruses. Also looking at the other terms in the ontology associated with this term, this term stands out as the exception without the word 'vesicle'.

Although looking at the ontology I wonder why there is the child term: GO:1902591 single-organism membrane budding, it has no annotations and I wonder if this is really needed?

**Comment by ValWood** [comment]
Is there any reason why  the parent "organelle organization" cannot go under
"single-organism process" then a lot of intermediate terms could be removed. Is there an example of organelle organization that is multi organism?

Similary vesicle-mediated transport (single organism process?)
Probably lots of others where the entire branch can go under single-organism process and remove all of the specific single-organism terms that have sprung up.

Otherwise nearly every term  in these branches would need a single-organism version?

It seems like a "high-level" ontology issue....

**Comment by BarbaraCzub** [comment]
@hdrabkin please go ahead with the obsoletions, and if you could also change the name 'membrane budding' to 'vesicle budding from membrane', this will be very helpful.
Thank you!

**Comment by hdrabkin** [comment]
Done, done, and done.

**Comment by BarbaraCzub** [comment]
Thank you!

**Comment by ValWood** [comment]
I think some of these processes are really MF terms.

https://github.com/geneontology/go-ontology/issues/13024#issuecomment-316634820

These should just be 
"MF membrane bending" involved_in  "BP vesicle assembly"

this gets rid of  unnecessary proliferation

membrane bending involved in x vesicle assembly.

I think the same can be said of scission (its performed by a GTPase), and there is a precedent for specific GTPases for membrane scission steps.
GO:1990606 membrane scission GTPase motor activity

Maybe these changes should not be made until  a model of the known molecular functions  for the pathway are aligned in order. There is no point to put processes representing molecular function in the process ontology.

**Comment by dosumis** [comment]
Is this ticket now done?

@ValWood  - good points about organelle organization and whether bending is MF (not sure I'm convinced), but seem like business for other tickets.

**Comment by ValWood** [comment]
Yep its a tricky one. It seems to be a single activity, but may require more than one gene product. One for the future....

**Comment by BarbaraCzub** [comment]
I am with @ValWood on the MF! At least PICALM Q13492 certainly seems to be bending the membrane on its own, as shown in PMID:25898166. Anyhow, this is a topic for other tickets indeed. I'll close this one now.

Thanks for all your comments,
Barbara

**Comment by ValWood** [comment]
Maybe this is a good topic for the meeting Weds afternoon session? How to distinguish a "Molecular Function" from a "Biological Process".

Increasingly  this blurry distinction seems to hinder us,  and causes a lot of duplication in the ontology, and a lot of work generally for ontology editors and curators, and a lot of annotation inconsistency, so it would be useful to address.

We have: 
a)  Single step functions described as processes, like membrane bending activity, glucose sensing, MF "protein anchor" vs  BP "cortical protein anchoring"  scission,  etc. etc. as well as the obvious single step molecular function  relationships between  protein modifier MFs and the BP corollary.

b) Many "MF involved in BP" terms in the BP ontology.
 This is *clearly unsustainable* to include everything precomposed here (personally would need to submit 1000's of  new terms, so we have been doing this with MF part_of B extensions to mirror Noctua modelling) . Is it useful to retain precomposed  MF involved in BP terms in  the MF ontology? ( they are unsustainable, confusing for newer curators and community curators and they make the annotation look inconsistent).

So maybe the objective primarily would be "Distinguishing MF from BP". I have never seen any substantial ontology guidelines for this (other than the vague multi-step vs. single step, but this gets confusing  when multiple genes are involved in an activity or it isn't a canonical function... 

Ontology editors appear to add requests for  terms into the BP ontology that sometimes seem like MF's  but if there was a checklist for the curator to follow along the lines:

Do you know the molecular activity of this protein: Y/N
If "Y" Does "MF part_of BP extension capture the contribution of the MF to the process: Y/N
If "Y" Do not request a BP term

I think maybe this could be a big bang-for-buck discussion
@thomaspd @ukemi  @pgaudet @hattrill 

A lot of the tickets appear to relate directly or indirectly to this issue.

**Comment by hattrill** [comment]
I agree that we could do with clarifying what constitutes a MF - if we move to a more function-centric view we will need to fill holes in MF or we will struggle. Many potential MFs are sequestered in BP....membrane bending, for example.....and sometimes there is either a lack of mechanistic knowledge (with small GTPases we rarely, if ever, capture the business end of things, we can only capture the function that turns them on or off).
Perhaps a Wednesday workshop would be good idea - get people to really think about the issue.
